Defect/de5920 dark theme

zarahzachz opened the pull request:
Fix some extraneous bugs.

- Add version number to footer (I feel like making this dynamic should be another story)
- Fix contrast in scss code example
    - `/developers/learn-more/css-sass`
- Clean up media queries on the nav so it works on all screen widths
    - Adding the fourth section for content admins threw off the flow of the nav
- Add some space in the `timepicker` example
    - `/ui-components/atoms/forms/timepicker`
- Update the theme button to extend to `iframe` code examples

https://github.com/crdschurch/crds-styles/pull/365

galiotoweb commented:
Hey @zarahzachz ! Looks great. Just a couple things.

I noticed you have crds-styles commits, but I don't see a crds-styles PR for those commits.

Also, I noticed the `.mobile-nav-trigger` in the navbar has a white background applied to it, so on dark theme it turns in a white icon on a white background. I think `.dark-theme .mobile-nav-trigger` just needs a transparent background

zarahzachz commented:
@galiotoweb Good catch! I forgot I updated the navbar styles in crds-styles. Didn't even notice the menu icon looking goofy... Updated and ready for review.
